README.md

```diff
@@ -1954,6 +1954,7 @@ Update Time, five active automations, webhooks.
 
 ## Other Free Resources
 
+  * [Aye.so](https://aye.so) - Design your link-in-bio in bento style using widgets. You can create a cool page using the widgets like contact form, images, subscribe, job post, and gifs.
```

Comment:
Website does not respond, I also just dont think it really fits the target audience as stated 

Thanks, but will close this now.